pkg/fsutils/usergroupadd_linux.go

```diff
@@ -1,4 +1,4 @@
-package idtools
+package fsutils
```

Comment:
This is the part of the package that I thought would move to `pkg/system`. It doesn't have much to do with files.

This file could move to `pkg/system/user/usergroupadd_linux.go`